### ISSUE TITLE
mprof executable doesn't work for me

### ISSUE THREAD

**Issue by patelamol** [pythonprofilers/memory_profiler#133]
Hi Whenever I try to do the 'mprof run <filename.py> ' it says 'mprof is not recognized as an internal or external command'.
But I am able to use the 'python -m memory_profiler <filename.py> ' but not the executable. Do you know why?

**Comment by fabianp** [comment]
how have you installed mprof?

**Comment by JoostJM** [comment]
@fabianp, I recieved the same error (I'm using windows 7), I think it is due to the fact that mprof does not have an extension and as such windows doesn't know what to do with it.

I fixed this by using this command:
`python C:\Python27\Scripts\mprof run <python-script-to-test>`

A possible fix for this is to specify `mprof` not as a script, but as an entrypoint. However, this does require you to provide a function, not a script (e.g. by creating a `main()` function in `mprof`). On windows, using the entry point results in the creation of an .exe launcher being created, which is placed in the scripts folder and therefore recoginzed directly from the command line (given that the scripts folder is included in the PATH environment variable).

If you'd like, I can create a pull request implementing this.
